src/components/TimelineEvent.js

```diff
@@ -2,9 +2,17 @@ import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
-  // Fill in your code here
-  return;
-}
+const TimelineEvent = (props) => {
+
+  const time = <Timestamp time={ props.timeStamp} />
```

Comment:
Good idea to make use of the Timestamp component and saving its return value in a variable